constance/templates/admin/constance/includes/results_list.html

```diff
@@ -20,7 +20,10 @@
             {{ item.form_field.errors }}
             {{ item.form_field }}
             <br>
-            <a href="#" onClick="document.getElementById('{{ item.form_field.auto_id }}').value = '{{ item.default|escapejs }}'; return false;">Reset to default</a>
+            <a href="#" onClick="
+              document.getElementById('{{ item.form_field.auto_id }}').{% if item.form_field_name == 'CheckboxInput' %}checked =
+                  {% if item.default == 'True' %} true {% else %} false {% endif %}
```

Comment:
I'm not very comfortable with the string comparison, but `item.default` is not a Boolean, but a String since it is passed through `localize` in [line 175](https://github.com/jazzband/django-constance/blob/81b723c91b7b1e708368c6d7cd04dc54e0abb878/constance/admin.py#L175):
```python
'default': localize(default),
```